.hlint.yaml

```diff
@@ -10,7 +10,7 @@
 - extensions:
   - default: false
   - name: [DeriveDataTypeable, GeneralizedNewtypeDeriving, NoMonomorphismRestriction, OverloadedStrings]
-  - name: [MultiWayIf, PatternGuards, RecordWildCards, ViewPatterns, TupleSections, LambdaCase]
+  - name: [MultiWayIf, PatternGuards, RecordWildCards, ViewPatterns, PatternSynonyms, TupleSections, LambdaCase]
```

Comment:
I've whitelisted `PatternSynonyms` so I can get CI further along.
If there is some issue with the extension I'll remove this commit and rewrite the code using them to use `View'` instead.

Reply:
Pattern synonyms are absolutely fine.